app/router.js

```diff
@@ -12,8 +12,8 @@ Router.map(function() {
     this.route('survey', function() {
       this.route('results');
       this.route('consent');
-      this.route('complete');
     });
+    this.route('complete', { path: '/survey/complete'});
```

Comment:
As discussed in person: 
1. The path to the participate route is `/`, so the expected url for the study completed page is `http://localhost:4200/survey/complete`.
2. The infinite loading behavior happens even when this feature is turned off (`featureFlags.continueSession = false`) so it is caused by something else